Sends emails when rent is created

Drocheg opened the pull request:
-Adds .env with environment variables
-Adds secrets from config/secrets.yml as environment variables
-Adds mail trap configuration
-Adds mailer to send the email when rent is created
-Adds sidekiq gem
-Adds dotenv gem

Drocheg committed:
Sends emails when rent is created

-Adds .env with environment variables
-Adds secrets from config/secrets.yml as environment variables
-Adds mail trap configuration
-Adds mailer to send the email when rent is created
-Adds sidekiq gem
-Adds dotenv gem

Drocheg commented:
https://trello.com/c/MltS60dh/11-mandar-email-al-usuario-cuando-se-crea-un-rent

Gmarn commented on app/controllers/api/v1/rents_controller.rb:
```diff
@@ -13,12 +13,15 @@ def create
         @rent = Rent.new(create_params_sanitized)
 
         if @rent.save
+          RentMailer.new_rent_notification(@rent.id).deliver_later
```
I recently realize, the `@` here is useless here for define the object rent. Any doubt on why ask me.

Gmarn commented on config/secrets.yml:
```diff
@@ -19,6 +19,14 @@
 
 development:
   secret_key_base: aca23d1befc693835c554a57e4b980ca3cd1d74ea145e07fea6357e1038c672e748e4201bf427ef889053eb56436a76e145dc0167dd17ca55c6a841097d9cb9a
+  mailer_domain: <%= ENV["MAILER_DOMAIN"] %>
```
Here, it is not important but do not forgot to add this at least at you production environment. You can use deftault env or import ones from dev in prod

Gmarn commented on app/views/rent_mailer/new_rent_notification.html.erb:
```diff
@@ -0,0 +1 @@
+<h1>R</h1>
```
You can delete this file if you are not using it.

Drocheg committed:
Fix mailing errors

-Adds mailing secrets in production, deletes
-Deletes 'new_rent_notification.html.erb' that was not used
-changes rent variable in rent controller from instance variable to local because it was only used locally.
